src/modules/style.ts

```diff
@@ -9,6 +9,7 @@ export type VNodeStyle = Record<string, string> & {
 // Binding `requestAnimationFrame` like this fixes a bug in IE/Edge. See #360 and #409.
 const raf =
   (typeof window !== "undefined" &&
+    typeof window.requestAnimationFrame === "function" &&
```

Comment:
does typescript have the optional chaining feature yet? If so, this might be cleaner:
```typescript
const raf = (typeof window?.requestAnimationFrame === "function" && …
```

Reply:
okay :) and there is a precedent for using that way as well
```bash
snabbdom水$ grep -ir \?\\. ./src/
./src/thunk.ts:  const ns = thunk.data?.ns;
./src/init.ts:  const isSameIs = vnode1.data?.is === vnode2.data?.is;
./src/init.ts:    const key = children[i]?.key;
./src/init.ts:      const init = data.hook?.init;
./src/init.ts:        hook.create?.(emptyNode, vnode);
./src/init.ts:    } else if (options?.experimental?.fragments && vnode.children) {
./src/init.ts:      data?.hook?.destroy?.(vnode);
./src/init.ts:          const removeHook = ch?.data?.hook?.remove;
./src/init.ts:    const hook = vnode.data?.hook;
./src/init.ts:    hook?.prepatch?.(oldVnode, vnode);
./src/init.ts:      vnode.data?.hook?.update?.(oldVnode, vnode);
./src/init.ts:    hook?.postpatch?.(oldVnode, vnode);
```

just a moment and I'll update this PR

Reply:
it is updated to look this way
```javascript
// Binding `requestAnimationFrame` like this fixes a bug in IE/Edge. See #360 and #409.
const raf = typeof window?.requestAnimationFrame === "function"
  ? window.requestAnimationFrame.bind(window)
  : setTimeout
```